pkgs/applications/window-managers/sway/default.nix

```diff
@@ -2,23 +2,24 @@
 , meson, ninja, pkg-config, wayland, scdoc
 , libxkbcommon, pcre, json_c, dbus, libevdev
 , pango, cairo, libinput, libcap, pam, gdk-pixbuf, librsvg
-, wlroots, wayland-protocols
+, wlroots, wayland-protocols, libdrm
 }:
 
 stdenv.mkDerivation rec {
   pname = "sway-unwrapped";
-  version = "1.5.1";
+  version = "1.6-rc1";
 
   src = fetchFromGitHub {
     owner = "swaywm";
     repo = "sway";
     rev = version;
-    sha256 = "1xsa3h8zhf29p0mi90baxpr76jkd9pd1gr97ky8cnjbcs4isj9j0";
+    sha256 = "0y3ycrvz72z8frvan12hv831v89vn8hx5v21q7w1y94izn8b08kk";
   };
 
   patches = [
     ./sway-config-no-nix-store-references.patch
     ./load-configuration-from-etc.patch
+    ./sway-allow-wlroots-13.patch
```

Comment:
Is this a Nixpkgs-only patch?

Reply:
Yes, but it will also be squashed away

Reply:
Such version ranges changes should use substituteInPlace.

Reply:
Do this via scripting, it will be easier to undo later.